lms/static/scripts/frontend_apps/components/test/LMSGrader-test.js

```diff
@@ -84,61 +90,73 @@ describe('LMSGrader', () => {
     );
   });
 
-  it('passes a default value of "0" to onChangeSelectedUser when no a student is selected', () => {
+  it('does not set a focus user by default', async () => {
     renderGrader();
-    assert.isTrue(fakeOnChange.calledWithExactly('0'));
+    await fakeSidebarWindow;
+    assert.isTrue(
+      fakeRpcCall.calledWith(
+        'fake window',
+        'fake origin',
+        'changeFocusModeUser',
+        [
+          {
+            username: undefined,
+            displayName: undefined,
+          },
+        ]
+      )
+    );
   });
-
-  it('passes the unique userid to onChangeSelectedUser when a student is selected', () => {
+  it('calls rpcCall to focus on that user when onSelectStudent is called with a valid student index', async () => {
     const wrapper = renderGrader();
-    act(() => {
-      wrapper
-        .find(FakeStudentSelector)
-        .props()
-        .onSelectStudent(1); // second student
-    });
-    wrapper.update();
-    assert.isTrue(fakeOnChange.calledWithExactly('student1'));
-  });
-
-  it('does not set a focus user by default', () => {
-    renderGrader();
-    sinon.assert.calledWith(fakeRemoveClientConfig, sinon.match(['focus']));
-  });
 
-  it('does not set a focus user when the user index is invalid', () => {
-    const wrapper = renderGrader();
     act(() => {
       wrapper
         .find(FakeStudentSelector)
         .props()
-        .onSelectStudent(-2); // invalid choice
+        .onSelectStudent(0); // note: initial index is -1
     });
-    wrapper.update();
 
-    sinon.assert.calledWith(fakeRemoveClientConfig, sinon.match(['focus']));
+    await fakeSidebarWindow;
+
+    assert.isTrue(
+      fakeRpcCall.calledWith(
+        'fake window',
+        'fake origin',
+        'changeFocusModeUser',
+        [
+          {
+            username: fakeStudents[0].userid,
+            displayName: fakeStudents[0].displayName,
+          },
+        ]
+      )
+    );
   });
 
-  it('changes the sidebar config to focus to the specified user when onSelectStudent is called with a valid user index', () => {
+  it('does not set a focus user when the user index is invalid', async () => {
     const wrapper = renderGrader();
     act(() => {
       wrapper
         .find(FakeStudentSelector)
         .props()
-        .onSelectStudent(0); // initial index is -1
+        .onSelectStudent(0); // invalid choice
```

Comment:
I'm a little confused as to why this _wasn't_ an invalid choice in the previous test?

Reply:
ahh, this should be -1

I think this needs to be `fakeRpcCall.secondCall` need to double check other tests here